kuka-driver/kuka_driver.cc

```diff
@@ -106,131 +203,114 @@ class KukaLCMClient : public KUKA::FRI::LBRClient {
 
   virtual void monitor() {
     KUKA::FRI::LBRClient::monitor();
-    PublishStateUpdate();
-  }
+    lcm_client_->UpdateRobotState(robot_id_, robotState());
```

Comment:
hmm, does robotState() return a arm status obj will all the correct numbers in it? 
i am guessing this is a function in iiwa API, and it returns a mutable reference?